packages/block-editor/src/hooks/style.js

```diff
@@ -251,6 +248,12 @@ export function addSaveProps(
 		}
 	} );
 
+	// @TODO
+	// Experimental notes:
+	// Maybe the style engine could do the classname work of `addSaveProps` in packages/block-editor/src/hooks/color.js here?
+	// And addSaveProps in packages/block-editor/src/hooks/font-family.js
+	// And add it to props.className?
+
```

Comment:
That makes good sense to me! As you mention further down, though, we could look at that in a separate PR if it winds up causing this one to grow too much.